e2e/Utils.ts

```diff
@@ -80,10 +81,10 @@ export const writeFiles = (
     const filename = filePath.pop(); // filepath becomes dirPath (no filename)
 
     if (filePath.length) {
-      createDirectory(path.join.apply(path, [directory].concat(filePath)));
+      createDirectory(path.join(directory, ...filePath));
     }
     fs.writeFileSync(
-      path.resolve.apply(path, [directory].concat(filePath, [filename])),
+      path.resolve(directory, ...filePath, filename || ''),
```

Comment:
```suggestion
      path.resolve(directory, fileOrPath),
```

then I think we can ditch `filename`? not sure. This should be fine though, you got rid of the ignore 🙂